ci/rpm_test_daos_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+# DAOS_PKG_VERSION environment variable needs to be set for this script
+
+if git show -s --format=%B | grep "^Skip-test: true"; then
+  exit 0
+fi
+nodelist=(${NODELIST//,/ })
+src/tests/ftest/config_file_gen.py -n ${nodelist[0]} \
+  -a /tmp/daos_agent.yml -s /tmp/daos_server.yml
+src/tests/ftest/config_file_gen.py -n $nodelist -d /tmp/dmg.yml
```

Comment:
(lint) Expanding an array without an index only gives the first element. [SC2128]
(lint) Double quote to prevent globbing and word splitting. [SC2086]